src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -499,23 +492,16 @@ const SecondHop = ({
     [swapperName, tradeQuote.recommendedSlippage],
   )
 
-  const networkFeeFiatPrecision = useMemo(
-    () => getHopTotalNetworkFeeFiatPrecision(tradeQuoteStep),
-    [tradeQuoteStep],
-  )
-
-  const protocolFeeFiatPrecision = useMemo(
-    () => getHopTotalProtocolFeesFiatPrecision(tradeQuoteStep),
-    [tradeQuoteStep],
-  )
+  const networkFeeFiatPrecision = selectHopTotalNetworkFeeFiatPrecision(store.getState(), 2)
+  const protocolFeeFiatPrecision = selectHopTotalProtocolFeesFiatPrecision(store.getState(), 2)
```

Comment:
second hop, so index is 1
```suggestion
  const networkFeeFiatPrecision = selectHopTotalNetworkFeeFiatPrecision(store.getState(), 1)
  const protocolFeeFiatPrecision = selectHopTotalProtocolFeesFiatPrecision(store.getState(), 1)
```

Reply:
https://github.com/shapeshift/web/pull/4782/commits/ff03b92e29699e757281d5948db6e783cb48ca52